quarkus/server/pom.xml

```
@@ -2,429 +2,58 @@
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
          xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <artifactId>keycloak-quarkus-parent</artifactId>
         <groupId>org.keycloak</groupId>
         <version>11.0.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
+    
+    <artifactId>keycloak-server</artifactId>
```

```suggestion
    <name>Keycloak Quarkus Server App</name>
    <artifactId>keycloak-quarkus-server-app</artifactId>
```

Not sure about the name, but would suggest keycloak-quarkus-server-app or keycloak-quarkus-server-launcher. We already have artifact with keycloak-server so needs to be something different.